Jellyfin.Plugin.Anime/Providers/AniDB/Metadata/AniDbMovieImageProvider.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Common.Net;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Entities.Movies;
+using MediaBrowser.Controller.Providers;
+using MediaBrowser.Model.Entities;
+using MediaBrowser.Model.Providers;
+
+namespace Jellyfin.Plugin.Anime.Providers.AniDB.Metadata
+{
+    /// <summary>
+    /// Provides movie images fetched from AniDB. Interally, this class uses AniDbSeriesImagesProvider
+    /// as AniDB does not differentiate between shows and movies.
+    /// </summary>
+    public class AniDbMovieImageProvider : IRemoteImageProvider
+    {
+        public string Name => "AniDB";
+        private readonly AniDbSeriesImagesProvider _seriesImagesProvider;
+
+        public AniDbMovieImageProvider(IApplicationPaths appPaths, IHttpClient httpClient)
+        {
+            _seriesImagesProvider = new AniDbSeriesImagesProvider(httpClient, appPaths);
+        }
+
+        public Task<HttpResponseInfo> GetImageResponse(string url, CancellationToken cancellationToken)
+        {
+            return _seriesImagesProvider.GetImageResponse(url, cancellationToken);
+        }
+
+        public async Task<IEnumerable<RemoteImageInfo>> GetImages(BaseItem item, CancellationToken cancellationToken)
+        {
+            var movie = (Movie)item;
+            var seriesId = movie.ProviderIds.GetOrDefault(ProviderNames.AniDb);
+            if (string.IsNullOrEmpty(seriesId))
+            {
+                return Enumerable.Empty<RemoteImageInfo>();
+            }
+
+            return await _seriesImagesProvider.GetImages(seriesId, cancellationToken);
+        }
+
+        public IEnumerable<ImageType> GetSupportedImages(BaseItem item)
+        {
+            return _seriesImagesProvider.GetSupportedImages(item);
+        }
+
+        public bool Supports(BaseItem item)
+        {
+            return item is Movie;
```

Comment:
```suggestion
            return item is Movie || item is Season || item is Series;
```

Reply:
I actually just checked the AniDB image providers, it looks like all of them use the exact same code. Since this interface doesn't use any generics you could probably combine them into a single `AniDbImageProvider.cs` class with the change I made above. If it doesn't work then disregard everything I said.

Reply:
It seems to work, so I've pushed the change. Anidb doesn't seem to provide any images other than the primary image, so while it looks weird that a show has the same image for the poster and all seasons, that's the only option when using AniDB as the only source.